src/main/java/com/rakovets/course/java/core/practice/operators/Task04.java

```diff
@@ -35,6 +34,7 @@ public static void main(String[] args) {
     static float getDistance(float milliseconds) {
         //TODO
         // Код, решающий задачу пишем ниже, при этом используя параметры метода
-        return 0.0f;
+
+        return milliseconds / 1000 * SOUND_SPEED * 1000 / 3600;
```

Comment:
"Code smell: Magic Number
Solution: Named Constants"

Reply:
Like this?
static final double GRAVITATIONAL_CONSTANT = 9.81;

double potentialEnergy(double mass, double height) {
  return mass * height * GRAVITATIONAL_CONSTANT;
}